scene/gui/tree.cpp

```diff
@@ -2447,6 +2449,14 @@ void Tree::_gui_input(Ref<InputEvent> p_event) {
 		}
 	}
 
+  if (drag_exited) {
```

Comment:
is there a reason why this cant be like

if ( condition_for_dragging && ( p_what== NOTIFICATION_WM_FOCUS_IN ||  p_what== NOTIFICATION_WM_FOCUS_OUT)) {
     disable dragging?
}

Reply:
p_what is a parameter for _notification function, and i think to cache the event in a variable is needed.  But i might be wrong..